### ISSUE TITLE
how can we use quadratic_weighted_kappa as a loss function in keras

### ISSUE THREAD

**Issue by mamro** [fchollet/keras#6294]
I need to use a custom quadratic_weighted_kappa loss function while training the model and as keras default loss does not contain this loss function 

I want to know how can I implement it within keras

Thanks
